src/sql/workbench/contrib/welcome/page/browser/welcomePage.css

```diff
@@ -144,28 +144,7 @@
 	justify-content: flex-start
 }
 
-.ads_homepage.XS:not(.SM) .btn_container div {
-	width: 100%;
-	width: 100%;
-}
-
-.ads_homepage.XS:not(.SM) .btn {
-	box-sizing: border-box;
-	width: 100%;
-	height: 34px;
-	padding-top: 1px;
-	font-size: 1.08em;
-	line-height: 29px;
-	border-radius: 2px;
-	display: flex;
-	align-content: center;
-	justify-content: center;
-	text-align: center;
-	text-decoration: none;
-	margin: 10px 0
-}
-
-.ads_homepage.SM .btn {
+/* .ads_homepage.SM .btn {
```

Comment:
Just delete if you don't need this anymore

Reply:
removed